unity/acsShowcase/Assets/Scripts/Calling/JoinTeamsMeeting.cs

```diff
@@ -64,7 +64,14 @@ public void Join()
             {
                 try
                 {
-                    CurrentCall = await currentCallAgent.JoinAsync(teamLocator, joinCallOptions);
+                    if (CallAgent is TeamsCallAgent teamsCallAgent)
+                    {
+                        CurrentCall = await teamsCallAgent.JoinAsync(teamLocator, joinCallOptions);
+                    }
+                    else if (CallAgent is CallAgent acsCallAgent)
+                    {
+                        CurrentCall = await acsCallAgent.JoinAsync(teamLocator, joinCallOptions);
+                    }
```

Comment:
add an else if with an error message:

`Log.Error<JoinTeamsMeeting>($"Unable to join call/meeting. Unknown call agent type {typeof(CallAgent)}.");`